data/json/generic_pair.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "type": "generic_pair",
+    "require": "thunder_effect",
+    "event": "thunder"
```

Comment:
Once you have a good new name for event, `generic_pair` should probably also be renamed.  Maybe `trigger_foo_on_requirements` (where `foo` is the new name for event) or something like that?